.circleci/config/jobs/push_operator_to_community_operators.yml

```diff
@@ -51,43 +51,23 @@ steps:
   - run:
       name: Push new Operator to the fork of the community-operators repository
       command: |
-        set -xeo pipefail
-
         CURRENT_DIRECTORY=$(pwd)
-        COMMUNITY_OPERATORS_UPSTREAM_LOCATION="${CURRENT_DIRECTORY}/community-operators"
-        DEPLOY_LOCATION="${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}/community-operators"
-        OPERATOR_LOCATION="${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator"
-
-        # Configure git user and gpg key
-        echo "${OPENSHIFT_OPERATOR_SIGNING_KEY_BASE64}" | base64 -d | gpg --import
-        git config --global commit.gpgsign true
-        git config --global user.signingkey "${OPENSHIFT_OPERATOR_SIGNING_KEY_ID}"
-        git config --global user.email "${OPENSHIFT_OPERATOR_GITHUB_EMAIL}"
-        git config --global user.name "${OPENSHIFT_OPERATOR_GITHUB_NAME}"
-
-        # Clone Community Operators repo from Snyk
-        git clone https://github.com/snyk/community-operators.git $COMMUNITY_OPERATORS_UPSTREAM_LOCATION
-        cd "${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}"
-        git checkout -b snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
-
-        # Copy new release to branch
-        cp -r "${OPERATOR_LOCATION}/${NEW_OPERATOR_VERSION}" "${DEPLOY_LOCATION}/snyk-operator/."
-        cp "${OPERATOR_LOCATION}/snyk-operator.package.yaml" "${DEPLOY_LOCATION}/snyk-operator/."
-
-        # Create the signed commit and push
-        git add "${DEPLOY_LOCATION}/snyk-operator/*"
-        git commit -s -m "Upgrade snyk-operator to version ${NEW_OPERATOR_VERSION}"
-        git push --set-upstream origin --force snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
-
+        COMMUNITY_FOLDER_LOCATION="community-operators"
+        ./scripts/operator/push-operator-to-snyk-upstream.sh "${CURRENT_DIRECTORY}" "${COMMUNITY_FOLDER_LOCATION}"
+  - run:
+      name: Open a Pull Request to the operator-framework/community-operators repository for community-operators
+      command: |
+        COMMUNITY_FOLDER_LOCATION="community-operators"
+        python ./scripts/operator/raise_pr_to_community_operators_from_our_fork.py "${COMMUNITY_FOLDER_LOCATION}" "${NEW_OPERATOR_VERSION}"
```

Comment:
will we be notified on slack about that? ideally with the PR link

Reply:
Yep! In the command below `./scripts/slack/notify_success_operator_push.py "${NEW_OPERATOR_VERSION}" "${COMMUNITY_FOLDER_LOCATION}"` it will send a Slack message to us.
```python
   data = {
      'attachments':
      [
        {
          'color': '#7CD197',
          'fallback': 'Build Notification: ' + circle_build_url,
          'title': 'Snyk Operator Pushed to GitHub repo snyk/community-operators',
          'text': 'Branch *snyk/snyk-operator-v' + operator_version + '* is ready for publishing to the ' + upstream_community + '.\n' +
            'https://github.com/operator-framework/community-operators/compare/master...snyk:snyk/' + upstream_community + '/snyk-operator-v' + operator_version
        }
      ]
    }
```

Reply:
this is about us pushing the code, it doesn't open the PR itself I think (the text of this message at least says that). do we actually open the PR?